wire/core/Pageimage.php

```diff
@@ -1810,6 +1897,120 @@ protected function ___install($filename) {
 		}
 	}
 
+	
+	/**
+	 * Get verbose DebugInfo, optionally with individual options array, @horst
+	 * (without invoking the magic debug)
+	 * 
+	 * @return mixed: array | string
+	 * 
+	 */
+	public function getDebugInfo($options = array(), $returnAsString = true) {
+		static $depth = 0;
+		$depth++;
+		
+		$thumb = "<img src='{$this->url}' style='max-width:120px; max-height:120px; ".($this->width() >= $this->height() ? 'width:100px; height:auto;' : 'height:100px; width:auto;')."' alt='' />";
+		$info = $this->original ? array('original' => $this->original->basename, 'basename' => $this->basename) : array('original' => $this->basename, 'basename' => $this->basename);
+		$info = array_merge(array('thumb' => $thumb), $info, parent::__debugInfo());
+		unset($info['filesize']);
+		if(!isset($_SERVER['HTTP_HOST'])) unset($info['thumb']);
+		
+		$oSizer = new ImageSizer($this->filename, $options);
+		$osInfo = $oSizer->getImageInfo(true);
+		$finalOptions = $oSizer->getOptions();
+		
+		$info['suffix'] = $finalOptions['suffix'];
```

Comment:
This may be a bit opinionated, but I believe that one array definition utilising `array_merge()` (since we already have some initial values) could make the code more readable:

```
$info = array_merge($info, array(
    'suffix' => $finalOptions['suffix'],
    'extension' => $osInfo['extension'],
    'imageType' => $osInfo['info']['imageType'],
    // etc.
);
```